CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+## Version 4.0.0 (April 10, 2015)
+
+Breaking changes:
+
+- The default normalization is now NFC, not NFKC.
+- The `fix_text` function has some new options that perform more targeted
+  operations that are part of NFKC normalization, without requiring hitting all
+  your text with the huge mallet that is NFKC.
+
+New features:
+
+- Fixers for strange new forms of mojibake, including particularly clear cases
+  of mixed UTF-8 and Windows-1252.
+- New heuristics, so that ftfy can fix more stuff, while maintaining
+  approximately zero false positives.
+- The command-line tool trusts you to know what encoding your *input* is in,
+  and assumes UTF-8 by default. You can still tell it to guess with the `-g`
+  option.
+- The command-line tool can be configured with options, and can be used as a
+  pipe.
+- Recognizes characters that are new in Unicode 7.0.
+- `remove_unsafe_private_use` has been removed entirely, after two versions
+  of deprecation.
```

Comment:
The deprecated `fix_bad_encoding` has also been removed.  (Also, removal of deprecated things might be better categorized under "breaking changes" than "new features".)